openmetadata-service/src/main/java/org/openmetadata/service/search/elasticsearch/ElasticSearchClient.java

```diff
@@ -704,11 +691,18 @@ public Map<String, Object> searchLineageInternal(
       int downstreamDepth,
       String queryFilter,
       boolean deleted,
+      boolean dataQualityOnly,
       String entityType)
       throws IOException {
     Map<String, Object> responseMap = new HashMap<>();
     Set<Map<String, Object>> edges = new HashSet<>();
     Set<Map<String, Object>> nodes = new HashSet<>();
+    if (dataQualityOnly) {
```

Comment:
i think we can add a different endpoint here @TeddyCr that will be better, since the basic flow s completely different , we are anyway trying to handle Pipeline and entity in this

Reply:
Sounds good. I was debating with this as well, make sense.